### ISSUE TITLE
Change in namespace?

### ISSUE THREAD

**Issue by enchilada404** [gesellkammer/loristrck#3]
I'm on Windows 10 and Python 3.9.2 64 bit. I just pip installed Loristrck 1.2.3. Now this happens:

```
>>> import loristrck as lt
>>> samples, sr = lt.sndreadmono("/path/to/sndfile.wav")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'loristrck' has no attribute 'sndreadmono'
```

Reading through the code I saw that some functions has moved to loristrck.util. However, I then tried to modify 'loristrck_pack' to reflect this - I changed line 69, 86 and 91 from 'loristrck' to 'loristrck.util'. Here's the output from my modified version (I call it 'ltpack'):

```
E:\kodes\loristrckrelease123\bin>py ltpack "E:\snd\testswavmono.wav"
Traceback (most recent call last):
  File "E:\kodes\loristrckrelease123\bin\ltpack", line 91, in <module>
    loristrck.util.partials_save_matrix(selected_partials,
  File "C:\Program Files\Python39\lib\site-packages\loristrck\util.py", line 804, in partials_save_matrix
    mtx = partials_sample(tracks, dt=dt, maxactive=maxactive)
  File "C:\Program Files\Python39\lib\site-packages\loristrck\util.py", line 490, in partials_sample
    m = np.hstack(cols)
  File "<__array_function__ internals>", line 5, in hstack
  File "C:\Program Files\Python39\lib\site-packages\numpy\core\shape_base.py", line 344, in hstack
    return _nx.concatenate(arrs, 0)
  File "<__array_function__ internals>", line 5, in concatenate
ValueError: all the input arrays must have same number of dimensions, but the array at index 0 has 1 dimension(s) and the array at index 1 has 2 dimension(s)
```

The sound being analyzed is a standard mono 16 bit, 44100 hz wav file.

Any idea what's going on here?

**Comment by gesellkammer** [comment]
thanks for the report, will take a look

**Comment by enchilada404** [comment]
Alright, seems to be fixed in 1.3.0

Thanks!   👌

**Comment by gesellkammer** [comment]
Fixed in 1.4.0.
